core-libs/setup/recipes/b2b/config/default-b2b-occ-config.ts

```diff
@@ -1,20 +1,14 @@
 // We need this import for augmentation of OccEndpoints to pick up
 import { CartOccEndpoints } from '@spartacus/cart/base/occ';
-import { CheckoutOccEndpoints } from '@spartacus/checkout/occ';
 import { OccConfig } from '@spartacus/core';
 import { OrderOccEndpoints } from '@spartacus/order/occ';
 import { UserAccountOccEndpoints } from '@spartacus/user/account/occ';
 import { UserProfileOccEndpoints } from '@spartacus/user/profile/occ';
 
-// While it is not strictly required to define checkout endpoints in a separate `CheckoutOccEndpoints`
-// variable, type augmentation does require that this file imports `CheckoutOccEndpoints`.
-// A good way to make sure the `CheckoutOccEndpoints` import is not removed by mistake is to use
-// `CheckoutOccEndpoints` in the code.
-const defaultB2bCheckoutOccEndpoints: CheckoutOccEndpoints = {
-  setDeliveryAddress: 'orgUsers/${userId}/carts/${cartId}/addresses/delivery',
-  placeOrder: 'orgUsers/${userId}/orders?fields=FULL',
```

Comment:
Removed, as we already have these configs in the b2b checkout (`feature-libs/checkout/b2b/occ/config/default-occ-checkout-b2b-config.ts`)